src/main/java/Task.java

```diff
@@ -22,9 +22,6 @@ public Task(String description) {
 
     public Task(){};
 
-    /**
-     * Mark done task with X
-     */
     public String getStatusIcon() {
         return (isDone ? "[X]" : "[ ]");
```

Comment:
Try to avoid the usage of Magic literals. You can save  "[X]" and "[ ]" as some variable and return that instead.